### PR TITLE
Fixes docs search by pinning sphinx-rtd-theme

### PR DESCRIPTION
**Pull request by dkliban**
fixes: #3569

### CODE REVIEW

**Review by decko — APPROVED** [review]

**Comment by quba42** [comment]
Should this also be added to pulp_rpm? Other plugins? Is there a mechanism for handling this consistently?

File: doc_requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 -r requirements.txt
 plantuml
 sphinx
-sphinx-rtd-theme
+sphinx-rtd-theme<1.2.0
```
Comment (mdellweg) [inline doc_requirements.txt:4]: Dependabot will probably try updating this right away. We need a comment here to prevent us from accepting that blindly.

**Comment by ipanova** [comment]
@quba42 i started to submit PRs to the other repos but then realized that there are just too many, so I've created this post https://discourse.pulpproject.org/t/every-pulp-repo-that-is-using-sphinx-rtd-theme-needs-to-pin-it-to-1-2-0/775

**Comment by quba42** [comment]
Opened a PR for pulp_deb: https://github.com/pulp/pulp_deb/pull/722

Will merge once it turns green.

**Pull request merged**